src/main.cpp

```diff
@@ -111,19 +112,34 @@ int main(int argc, char **argv)
 	QFile f(path+"Preferences.ini");
 	f.open(QFile::ReadOnly);
 	QTextStream in (&f);
-	const QStringList content = in.readAll().split("\n");
+    const QStringList content = in.readAll().split("\n");
 	QString language="default";
-	for(auto s : content)
-	{
-		if(s.startsWith("language"))
-		{
-			language=s.split("=")[1];
+	for(auto s : content) {
+		if(s.startsWith("language")) {
+			language=s.split("=")[1].trimmed();
```

Comment:
From what I understand in this lines, the ini file, `Preferences.ini`, is manually opened and parsed. I think the language should be retrieved through the Preferences API.